packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -525,34 +527,40 @@ const Summary: React.FunctionComponent = () => {
 		const { isSiteTitleStepVisible, domain, planProductId } = launchStore.getState();
 
 		return [ launchStore.hasSelectedDomain(), isSiteTitleStepVisible, domain, planProductId ];
-	} );
+	}, [] );
 
-	const { isSelectedPlanFree } = useSelect( ( select ) => {
-		const plansStore = select( PLANS_STORE );
-
-		return {
-			isSelectedPlanFree: plansStore.isPlanProductFree( selectedPlanProductId ),
-		};
-	} );
+	const isSelectedPlanPaid = useSelect(
+		( select ) => !! select( LAUNCH_STORE ).getPaidPlanProductId(),
+		[]
+	);
```

Comment:
Should be refactored once #49930 is merged (see [this comment](https://github.com/Automattic/wp-calypso/pull/49930/files#r575511915) for context)

Reply:
Same here. See https://github.com/Automattic/wp-calypso/pull/48452#discussion_r576310104

Reply:
Resolving this comment, as it will be fixed in #49930 (see https://github.com/Automattic/wp-calypso/pull/49930#issuecomment-779856916)